templates/macros/assets.html

```diff
@@ -1,11 +1,16 @@
 {% import "macros/images.html" as image_macros %}
 
-{% macro license_logo_svg() %}
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512">
-    <!--! Font Awesome Pro 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
-    <path d="M554.9 154.5c-17.62-35.25-68.12-35.38-85.87 0c-87 174.3-84.1 165.9-84.1 181.5c0 44.13 57.25 80 128 80s127.1-35.88 127.1-80C639.1 319.9 641.4 327.3 554.9 154.5zM439.1 320l71.96-144l72.17 144H439.1zM256 336c0-16.12 1.375-8.75-85.12-181.5c-17.62-35.25-68.12-35.38-85.87 0c-87 174.3-84.1 165.9-84.1 181.5c0 44.13 57.25 80 127.1 80S256 380.1 256 336zM127.9 176L200.1 320H55.96L127.9 176zM495.1 448h-143.1V153.3C375.5 143 393.1 121.8 398.4 96h113.6c17.67 0 31.1-14.33 31.1-32s-14.33-32-31.1-32h-128.4c-14.62-19.38-37.5-32-63.62-32S270.1 12.62 256.4 32H128C110.3 32 96 46.33 96 64S110.3 96 127.1 96h113.6c5.25 25.75 22.87 47 46.37 57.25V448H144c-26.51 0-48.01 21.49-48.01 48c0 8.836 7.165 16 16 16h416c8.836 0 16-7.164 16-16C544 469.5 522.5 448 495.1 448z"/>
+{% macro init_svg() %}
+<svg style="display: none" version="2.0">
+    <defs>
+        <symbol id="asset-card__scale-balanced-svg" viewBox="0 0 640 512">
+            <!--! Font Awesome Pro 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
```

Comment:
Hmm this is explicitly calling itself out as "font awesome pro". We shouldn't be using any icons from there (and I dont think this one is part of pro).

Reply:
https://fontawesome.com/license/free

Reply:
That's the exact svg with the same comment I get when I copy it from there https://fontawesome.com/icons/scale-balanced?s=solid

I'm not sure why it says it's under the pro license

Reply:
If you download the "free" package, the svgs have the correct license embedded in them. I just pushed that svg content.

Reply:
Interesting, it's a bit weird that they have the wrong license on the site for the exact same svg. That's good to know though so I downloaded it for the future.